openmls/src/ciphersuite/mod.rs

```diff
@@ -301,6 +359,9 @@ impl PartialEq for Secret {
 impl Secret {
     /// Randomly sample a fresh `Secret`.
     /// This default random initialiser uses the default Secret length of `hash_length`.
+    ///
+    /// **NOTE: This has to wait until it can acquire the lock to get randomness!**
+    /// TODO: This panics if another thread holding the rng panics.
```

Comment:
Can you file an issue for this so we don't forget?

Reply:
I filed #467 to generally address the handling of randomness